src/RobotCommandsNetworker.cpp

```diff
@@ -85,9 +85,10 @@ rtt::RobotCommands protoToRobotCommands(const proto::RobotCommands& protoCommand
 // Blue commands publisher
 RobotCommandsBluePublisher::RobotCommandsBluePublisher() : utils::Publisher(utils::ChannelType::ROBOT_COMMANDS_BLUE_CHANNEL) {}
 
-bool RobotCommandsBluePublisher::publish(const rtt::RobotCommands& commands) {
+std::size_t RobotCommandsBluePublisher::publish(const rtt::RobotCommands& commands) {
     auto protoRobotCommands = robotCommandsToProto(commands);
-    return this->send(protoRobotCommands.SerializeAsString());
+
+    return this->send(protoRobotCommands.SerializeAsString()) ? protoRobotCommands.ByteSizeLong() : -1;
```

Comment:
-1 of std::size_t will just underflow: maybe prefer using std::numeric_types<std::size_t>::max() in that case? That is a bit cleaner as -1 is a conversion.